client/src/components/Navbar.js

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import logoImage from './../images'
+import {
+  Collapse,
+  Navbar,
+  NavbarToggler,
+  NavbarBrand,
+  Nav,
+  NavItem,
+  NavLink,
+  Dropdown,
+  DropdownToggle,
+  DropdownMenu,
+  DropdownItem, Button } from 'reactstrap';
+
+export default class BFSNav extends React.Component {
```

Comment:
nitpick (just stylistic things to keep in mind!): In your first line you can make it `import React, { Component } from 'react'` so you don't have to do React.Component here, and just replace it with `... extends Component {`